Sprint-3/quote-generator/quotes.js

```diff
@@ -489,5 +491,18 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
-
 // call pickFromArray with the quotes array to check you get a random quote
+function newquote(){
+let items = []
```

Comment:
This works, but there is no need to create a new empty array and them push quotes into the empty array. It’s inefficient because you are then keeping two copies of the same list. Try changing it to using only quotes :)

Reply:
Thank you now I can see that creating a new array and then pushing a single quote into it is unnecessary.
I have improved.